packages/vue3-vite/src/generators/vue3-vite/schema.d.ts

```diff
@@ -1,6 +1,9 @@
+import { PathAlias } from './path';
+
 export interface Vue3ViteGeneratorSchema {
   name: string;
   title?: string;
   tags?: string;
   directory?: string;
+  alias?: PathAlias;
```

Comment:
I added an `alias` option to the app generator, the purpose is to determine if the generated app should use local alias (use the `paths` defined in `apps/myapp/tsconfig.json`) or global alias (use the `paths` defined in the `tsconfig.base.json` in workspace root).

I'm not sure if there are official names for these "local alias" and "global alias".

Reply:
So if I run `npx nx g nx-vue3-vite:app --alias global`, it'll generate an app that:
- `baseUrl` and `paths` are NOT written in `tsconfig.json`, so that we can use the `paths` defined in `tsconfig.base.json`.
- The `resolve` in `vite.config.ts` will be:
  ```js
  resolve: {
    alias: {
      ...tsconfigBaseAliases(),
      // Add your alias here.
    }
  }
  ```

On the other hand, if I run `npx nx g nx-vue3-vite:app --alias local` (or omit the `alias` option), it'll generate an app that:
- `baseUrl` and `paths` are written in `tsconfig.json`.
- The `resolve` in `vite.config.ts` will be:
  ```js
  resolve: {
    alias: {
      '@assets/': `${path.resolve(__dirname, './src/assets')}/`,
      '@app/': `${path.resolve(__dirname, './src/app')}/`,
      '@public/': `${path.resolve(__dirname, './src/public')}/`,
    }
  }
  ```

Reply:
I think the naming is fine! I'll add some more description in the README/schema about what it means.

The behavior seems fine! The only change I'd like to make is to set `global` as the default. The reason is that this is the "nx" way of setting up a project, and using local aliases is not their recommended approach. Hopefully in the future we can find a way of supporting both, though.